solaris/SolarisProcessList.h

```diff
@@ -13,8 +13,6 @@ in the source distribution for its full text.
 #define GZONE "global    "
 #define UZONE "unknown   "
```

Comment:
Any reason those constants are above the includes? Documentation would be good for this.

Reply:
I am unsure why those constants are above the includes. But I think defining them after includes won't break anything as ```GZONE``` and ```UZONE``` are passed to ```xstrdup()``` .